src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -42,8 +42,8 @@ public Auth deserialize(JsonElement json, Type typeOfT, JsonDeserializationConte
 
   private final String clientId;
   private final String clientSecret;
-  private final String redirectUri;
-  private final boolean testMode;
+  private final String redirectUri;    
```

Comment:
```suggestion
  private final String redirectUri;
```
nit: trailing whitespace